flask-server/migrations/versions/e11b182cd2e2_updated_table_and_column_names_for_.py

```diff
@@ -0,0 +1,154 @@
+"""Updated table and column names for content tables to be lowercase
```

Comment:
Very diligent to make these!
In a production environment it's very handy to have the last upgraden en downgrade at hand for releasing and potential rollbacks if needed.